doc/_scripts/redirects.py

```diff
@@ -173,6 +173,8 @@
     ('reference/util/index', 'kernel/util/index'),
     ('samples/drivers/kscan_touch', 'samples/subsys/input/input'),
     ('samples/net/cloud/google_iot_mqtt', 'samples/net/cloud'),
+    ('samples/sensor/wsen_hids', 'samples/sensor'),
+    ('samples/sensor/wsen_itds', 'samples/sensor'),
```

Comment:
sorry, didn't realize it was not correct:

```suggestion
    ('samples/sensor/wsen_hids/README', 'samples/sensor/sensor'),
    ('samples/sensor/wsen_itds/README', 'samples/sensor/sensor'),
```

if you feel like fixing the line above, in a separate commit, this would be appreciated:

```
    ('samples/net/cloud/google_iot_mqtt/README', 'samples/net/cloud/cloud'),
```